src/protocols/secure_channel/tests/TestCASESession.cpp

```diff
@@ -316,7 +321,7 @@ class TestCASESessionPersistentStorageDelegate : public PersistentStorageDelegat
                 keysize[i] = static_cast<uint16_t>(strlen(key));
                 keysize[i]++;
                 keys[i] = reinterpret_cast<char *>(chip::Platform::MemoryAlloc(keysize[i]));
-                strcpy(keys[i], key);
+                memcpy(keys[i], key, keysize[i]);
```

Comment:
Keysize would be a character shorter in that case since strlen doesn't count the null character. I suggest keeping strcpy or increasing the keysize by 1 to keep a valid string.

Reply:
nevermind, didn't see the `keysize[i]++;`

Reply:
> Keysize would be a character shorter in that case since strlen doesn't count the null character.

Yes, "keys[i]" does not include the null char.

>  I suggest keeping strcpy 

It's misleading, because it actually runs out of buffer and just does not put in the null... So the behavior with memcpy is the same, but clearer in terms of what's going on.